util/codebuild/codebuild-prowler-audit-account-cfn.yaml

```diff
@@ -141,7 +141,7 @@ Resources:
           - id: W28
             reason: "Explicit name is required for this resource to avoid circular dependencies."
     Properties:
-      RoleName: !Sub 'prowler-codebuild-role'
+      RoleName: 'prowler-codebuild-role'
```

Comment:
why this?

Reply:
It as failing for cfn-lint, as `!Sub` is useless with just static text. There is not variable to perform substitution.